### ISSUE TITLE
2025-05-23, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#178]
# Today's Arxiv Papers

(1) [SophiaVL-R1: Reinforcing MLLMs Reasoning with Thinking Reward](http://arxiv.org/abs/2505.17018v1), 2025-05-22

(2) [Let Androids Dream of Electric Sheep: A Human-like Image Implication Understanding and Reasoning Framework](http://arxiv.org/abs/2505.17019v1), 2025-05-22

(3) [CrossLMM: Decoupling Long Video Sequences from LMMs via Dual Cross-Attention Mechanisms](http://arxiv.org/abs/2505.17020v1), 2025-05-22

(4) [Delving into RL for Image Generation with CoT: A Study on DPO vs. GRPO](http://arxiv.org/abs/2505.17017v1), 2025-05-22

(5) [Multi-SpatialMLLM: Multi-Frame Spatial Understanding with Multi-Modal Large Language Models](http://arxiv.org/abs/2505.17015v1), 2025-05-22

(6) [SpatialScore: Towards Unified Evaluation for Multimodal Spatial Understanding](http://arxiv.org/abs/2505.17012v1), 2025-05-22

(7) [R1-Searcher++: Incentivizing the Dynamic Knowledge Acquisition of LLMs via Reinforcement Learning](http://arxiv.org/abs/2505.17005v1), 2025-05-22

(8) [Do Large Language Models Excel in Complex Logical Reasoning with Formal Language?](http://arxiv.org/abs/2505.16998v1), 2025-05-22

(9) [X-MAS: Towards Building Multi-Agent Systems with Heterogeneous LLMs](http://arxiv.org/abs/2505.16997v1), 2025-05-22

(10) [DecoupledESC: Enhancing Emotional Support Generation via Strategy-Response Decoupled Preference Optimization](http://arxiv.org/abs/2505.16995v1), 2025-05-22

(11) [$\text{R}^2\text{ec}$: Towards Large Recommender Models with Reasoning](http://arxiv.org/abs/2505.16994v1), 2025-05-22

(12) [Dimple: Discrete Diffusion Multimodal Large Language Model with Parallel Decoding](http://arxiv.org/abs/2505.16990v1), 2025-05-22

(13) [MASLab: A Unified and Comprehensive Codebase for LLM-based Multi-Agent Systems](http://arxiv.org/abs/2505.16988v1), 2025-05-22

(14) [T1: A Tool-Oriented Conversational Dataset for Multi-Turn Agentic Planning](http://arxiv.org/abs/2505.16986v1), 2025-05-22

(15) [UFT: Unifying Supervised and Reinforcement Fine-Tuning](http://arxiv.org/abs/2505.16984v1), 2025-05-22

(16) [LLM as Effective Streaming Processor: Bridging Streaming-Batch Mismatches with Group Position Encoding](http://arxiv.org/abs/2505.16983v1), 2025-05-22

(17) [Beyond Correlation: Towards Causal Large Language Model Agents in Biomedicine](http://arxiv.org/abs/2505.16982v1), 2025-05-22

(18) [Know the Ropes: A Heuristic Strategy for LLM-based Multi-Agent System Design](http://arxiv.org/abs/2505.16979v1), 2025-05-22

(19) [HyGenar: An LLM-Driven Hybrid Genetic Algorithm for Few-Shot Grammar Generation](http://arxiv.org/abs/2505.16978v1), 2025-05-22

(20) [Creatively Upscaling Images with Global-Regional Priors](http://arxiv.org/abs/2505.16976v1), 2025-05-22

(21) [SWE-Dev: Evaluating and Training Autonomous Feature-Driven Software Development](http://arxiv.org/abs/2505.16975v1), 2025-05-22

(22) [VeriFastScore: Speeding up long-form factuality evaluation](http://arxiv.org/abs/2505.16973v1), 2025-05-22

(23) [CASS: Nvidia to AMD Transpilation with Data, Models, and Benchmark](http://arxiv.org/abs/2505.16968v1), 2025-05-22

(24) [Fixing Data That Hurts Performance: Cascading LLMs to Relabel Hard Negatives for Robust Information Retrieval](http://arxiv.org/abs/2505.16967v1), 2025-05-22

(25) [MedFrameQA: A Multi-Image Medical VQA Benchmark for Clinical Reasoning](http://arxiv.org/abs/2505.16964v1), 2025-05-22

(26) [Invisible Prompts, Visible Threats: Malicious Font Injection in External Resources for Large Language Models](http://arxiv.org/abs/2505.16957v1), 2025-05-22

(27) [Cracking Aegis: An Adversarial LLM-based Game for Raising Awareness of Vulnerabilities in Privacy Protection](http://dx.doi.org/10.1145/3715336.3735812), 2025-05-22

(28) [A Comprehensive Evaluation of Contemporary ML-Based Solvers for Combinatorial Optimization](http://arxiv.org/abs/2505.16952v1), 2025-05-22

(29) [Bottlenecked Transformers: Periodic KV Cache Abstraction for Generalised Reasoning](http://arxiv.org/abs/2505.16950v1), 2025-05-22

(30) [MixAT: Combining Continuous and Discrete Adversarial Training for LLMs](http://arxiv.org/abs/2505.16947v1), 2025-05-22

(31) [AGENTIF: Benchmarking Instruction Following of Large Language Models in Agentic Scenarios](http://arxiv.org/abs/2505.16944v1), 2025-05-22

(32) [In-Context Watermarks for Large Language Models](http://arxiv.org/abs/2505.16934v1), 2025-05-22

(33) [LLaDA-V: Large Language Diffusion Models with Visual Instruction Tuning](http://arxiv.org/abs/2505.16933v1), 2025-05-22

(34) [Beyond Needle(s) in the Embodied Haystack: Environment, Architecture, and Training Considerations for Long Context Reasoning](http://arxiv.org/abs/2505.16928v1), 2025-05-22

(35) [UNCLE: Uncertainty Expressions in Long-Form Generation](http://arxiv.org/abs/2505.16922v1), 2025-05-22

(36) [Scalable and Interpretable Contextual Bandits: A Literature Review and Retail Offer Prototype](http://arxiv.org/abs/2505.16918v1), 2025-05-22

(37) [Backdoor Cleaning without External Guidance in MLLM Fine-tuning](http://arxiv.org/abs/2505.16916v1), 2025-05-22

(38) [Unsupervised Prompting for Graph Neural Networks](http://arxiv.org/abs/2505.16903v1), 2025-05-22

(39) [Code Graph Model (CGM): A Graph-Integrated Large Language Model for Repository-Level Software Engineering Tasks](http://arxiv.org/abs/2505.16901v1), 2025-05-22

(40) [Power-Law Decay Loss for Large Language Model Finetuning: Focusing on Information Sparsity to Enhance Generation Quality](http://arxiv.org/abs/2505.16900v1), 2025-05-22

(41) [Shadows in the Attention: Contextual Perturbation and Representation Drift in the Dynamics of Hallucination in LLMs](http://arxiv.org/abs/2505.16894v1), 2025-05-22

(42) [CAIN: Hijacking LLM-Humans Conversations via a Two-Stage Malicious System Prompt Generation and Refining Framework](http://arxiv.org/abs/2505.16888v1), 2025-05-22

(43) [Don't "Overthink" Passage Reranking: Is Reasoning Truly Necessary?](http://arxiv.org/abs/2505.16886v1), 2025-05-22

(44) [CASTILLO: Characterizing Response Length Distributions of Large Language Models](http://arxiv.org/abs/2505.16881v1), 2025-05-22

(45) [MPO: Multilingual Safety Alignment via Reward Gap Optimization](http://arxiv.org/abs/2505.16869v1), 2025-05-22

(46) [Walk&Retrieve: Simple Yet Effective Zero-shot Retrieval-Augmented Generation via Knowledge Graph Walks](http://arxiv.org/abs/2505.16849v1), 2025-05-22

(47) [R1-Compress: Long Chain-of-Thought Compression via Chunk Compression and Search](http://arxiv.org/abs/2505.16838v1), 2025-05-22

(48) [SimpleDeepSearcher: Deep Information Seeking via Web-Powered Reasoning Trajectory Synthesis](http://arxiv.org/abs/2505.16834v1), 2025-05-22

(49) [Unlearning Isn't Deletion: Investigating Reversibility of Machine Unlearning in LLMs](http://arxiv.org/abs/2505.16831v1), 2025-05-22

(50) [KTAE: A Model-Free Algorithm to Key-Tokens Advantage Estimation in Mathematical Reasoning](http://arxiv.org/abs/2505.16826v1), 2025-05-22

(51) [LLM-Based Emulation of the Radio Resource Control Layer: Towards AI-Native RAN Protocols](http://arxiv.org/abs/2505.16821v1), 2025-05-22

(52) [Action2Dialogue: Generating Character-Centric Narratives from Scene-Level Prompts](http://arxiv.org/abs/2505.16819v1), 2025-05-22

(53) [DeepRec: Towards a Deep Dive Into the Item Space with Large Language Model Based Recommendation](http://arxiv.org/abs/2505.16810v1), 2025-05-22

(54) [Two-way Evidence self-Alignment based Dual-Gated Reasoning Enhancement](http://arxiv.org/abs/2505.16806v1), 2025-05-22

(55) [Learning Beyond Limits: Multitask Learning and Synthetic Data for Low-Resource Canonical Morpheme Segmentation](http://arxiv.org/abs/2505.16800v1), 2025-05-22

(56) [Accidental Misalignment: Fine-Tuning Language Models Induces Unexpected Vulnerability](http://arxiv.org/abs/2505.16789v1), 2025-05-22

(57) [CoTSRF: Utilize Chain of Thought as Stealthy and Robust Fingerprint of Large Language Models](http://arxiv.org/abs/2505.16785v1), 2025-05-22

(58) [Reasoning Beyond Language: A Comprehensive Survey on Latent Chain-of-Thought Reasoning](http://arxiv.org/abs/2505.16782v1), 2025-05-22

(59) [IFEval-Audio: Benchmarking Instruction-Following Capability in Audio-based Large Language Models](http://arxiv.org/abs/2505.16774v1), 2025-05-22

(60) [When Safety Detectors Aren't Enough: A Stealthy and Effective Jailbreak Attack on LLMs via Steganographic Techniques](http://arxiv.org/abs/2505.16765v1), 2025-05-22

(61) [TRIM: Achieving Extreme Sparsity with Targeted Row-wise Iterative Metric-driven Pruning](http://arxiv.org/abs/2505.16743v1), 2025-05-22

(62) [Mitigating Fine-tuning Risks in LLMs via Safety-Aware Probing Optimization](http://arxiv.org/abs/2505.16737v1), 2025-05-22

(63) [Robust LLM Fingerprinting via Domain-Specific Watermarks](http://arxiv.org/abs/2505.16723v1), 2025-05-22

(64) [Breaking mBad! Supervised Fine-tuning for Cross-Lingual Detoxification](http://arxiv.org/abs/2505.16722v1), 2025-05-22

(65) [Training Long-Context LLMs Efficiently via Chunk-wise Optimization](http://arxiv.org/abs/2505.16710v1), 2025-05-22

(66) [Locate-then-Merge: Neuron-Level Parameter Fusion for Mitigating Catastrophic Forgetting in Multimodal LLMs](http://arxiv.org/abs/2505.16703v1), 2025-05-22

(67) [MCP-RADAR: A Multi-Dimensional Benchmark for Evaluating Tool Use Capabilities in Large Language Models](http://arxiv.org/abs/2505.16700v1), 2025-05-22

(68) [Software Architecture Meets LLMs: A Systematic Literature Review](http://arxiv.org/abs/2505.16697v1), 2025-05-22

(69) [Beyond Induction Heads: In-Context Meta Learning Induces Multi-Phase Circuit Emergence](http://arxiv.org/abs/2505.16694v1), 2025-05-22

(70) [Your Pre-trained LLM is Secretly an Unsupervised Confidence Calibrator](http://arxiv.org/abs/2505.16690v1), 2025-05-22

(71) [Hybrid Parameterized Quantum States for Variational Quantum Learning](http://arxiv.org/abs/2505.16676v1), 2025-05-22

(72) [R1-ShareVL: Incentivizing Reasoning Capability of Multimodal Large Language Models via Share-GRPO](http://arxiv.org/abs/2505.16673v1), 2025-05-22

(73) [BitHydra: Towards Bit-flip Inference Cost Attack against Large Language Models](http://arxiv.org/abs/2505.16670v1), 2025-05-22

(74) [ELABORATION: A Comprehensive Benchmark on Human-LLM Competitive Programming](http://arxiv.org/abs/2505.16667v1), 2025-05-22

(75) [A Japanese Language Model and Three New Evaluation Benchmarks for Pharmaceutical NLP](http://arxiv.org/abs/2505.16661v1), 2025-05-22

(76) [SD-MAD: Sign-Driven Few-shot Multi-Anomaly Detection in Medical Images](http://arxiv.org/abs/2505.16659v1), 2025-05-22

(77) [Seeing Far and Clearly: Mitigating Hallucinations in MLLMs with Attention Causal Decoding](http://arxiv.org/abs/2505.16652v1), 2025-05-22

(78) [Collaboration among Multiple Large Language Models for Medical Question Answering](http://arxiv.org/abs/2505.16648v1), 2025-05-22

(79) [SMART: Self-Generating and Self-Validating Multi-Dimensional Assessment for LLMs' Mathematical Problem Solving](http://arxiv.org/abs/2505.16646v1), 2025-05-22

(80) [From Evaluation to Defense: Advancing Safety in Video Large Language Models](http://arxiv.org/abs/2505.16643v1), 2025-05-22

(81) [SSR-Zero: Simple Self-Rewarding Reinforcement Learning for Machine Translation](http://arxiv.org/abs/2505.16637v1), 2025-05-22

(82) [Steering Large Language Models for Machine Translation Personalization](http://arxiv.org/abs/2505.16612v1), 2025-05-22

(83) [From Generic Empathy to Personalized Emotional Support: A Self-Evolution Framework for User Preference Alignment](http://arxiv.org/abs/2505.16610v1), 2025-05-22

(84) [What Media Frames Reveal About Stance: A Dataset and Study about Memes in Climate Change Discourse](http://arxiv.org/abs/2505.16592v1), 2025-05-22

(85) [Evaluating Large Language Model with Knowledge Oriented Language Specific Simple Question Answering](http://arxiv.org/abs/2505.16591v1), 2025-05-22

(86) [Beyond LLMs: An Exploration of Small Open-source Language Models in Logging Statement Generation](http://arxiv.org/abs/2505.16590v1), 2025-05-22

(87) [A Survey on the Application of Large Language Models in Scenario-Based Testing of Automated Driving Systems](http://arxiv.org/abs/2505.16587v1), 2025-05-22

(88) [O$^2$-Searcher: A Searching-based Agent Model for Open-Domain Open-Ended Question Answering](http://arxiv.org/abs/2505.16582v1), 2025-05-22

(89) [Bridging the Dynamic Perception Gap: Training-Free Draft Chain-of-Thought for Dynamic Multimodal Spatial Reasoning](http://arxiv.org/abs/2505.16579v1), 2025-05-22

(90) [Large Language Model-Empowered Interactive Load Forecasting](http://arxiv.org/abs/2505.16577v1), 2025-05-22

(91) [EMULATE: A Multi-Agent Framework for Determining the Veracity of Atomic Claims by Emulating Human Actions](http://arxiv.org/abs/2505.16576v1), 2025-05-22

(92) [URLs Help, Topics Guide: Understanding Metadata Utility in LLM Training](http://arxiv.org/abs/2505.16570v1), 2025-05-22

(93) [Finetuning-Activated Backdoors in LLMs](http://arxiv.org/abs/2505.16567v1), 2025-05-22

(94) [ScholarBench: A Bilingual Benchmark for Abstraction, Comprehension, and Reasoning Evaluation in Academic Contexts](http://arxiv.org/abs/2505.16566v1), 2025-05-22

(95) [CTRAP: Embedding Collapse Trap to Safeguard Large Language Models from Harmful Fine-Tuning](http://arxiv.org/abs/2505.16559v1), 2025-05-22

(96) [Is Your LLM-Based Multi-Agent a Reliable Real-World Planner? Exploring Fraud Detection in Travel Planning](http://arxiv.org/abs/2505.16557v1), 2025-05-22

(97) [Think Silently, Think Fast: Dynamic Latent Compression of LLM Reasoning Chains](http://arxiv.org/abs/2505.16552v1), 2025-05-22

(98) [Incremental Sequence Classification with Temporal Consistency](http://arxiv.org/abs/2505.16548v1), 2025-05-22

(99) [Mechanistic Understanding and Mitigation of Language Confusion in English-Centric Large Language Models](http://arxiv.org/abs/2505.16538v1), 2025-05-22

(100) [Causal-Invariant Cross-Domain Out-of-Distribution Recommendation](http://arxiv.org/abs/2505.16532v1), 2025-05-22
